File: VMS_Backend/VMS_Backend/Data/Models/CompanySettings.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace VMS_Backend.Data.Models
{
    [Table("company_settings")]
    public class CompanySettings
    {
        [Key]
        [Column("company_id")] 
        public int CompanyId { get; set; }
        
        [ForeignKey("CompanyId")]
        public Company Company { get; set; }
        
        [Column("android_interval_recording")]
        public int AndroidIntervalRecording { get; set; }
        
        [Column("android_interval_synchronization")]
        public int AndroidIntervalSynchronization { get; set; }
    }
}

File: VehicleMonitoringSystemWeb/vms_frontend/src/models/companySettings.ts
export default class CompanySettings {
  public companyId: number | undefined;
  public androidIntervalRecording: number | undefined;
  public androidIntervalSynchronization: number | undefined;

  constructor(companyId: number, androidIntervalRecording: number, androidIntervalSynchronization: number) {
    this.companyId = companyId;
    this.androidIntervalRecording = androidIntervalRecording;
    this.androidIntervalSynchronization = androidIntervalSynchronization;
  }
}
